File: 2_4/2task.cpp
#include <iostream>
#include <string>
#include <set>
using namespace std;
bool Pangram(const string& sentence) {
    set<char> letters;
    for (char ch : sentence) {
        if (isalpha(ch)) {
            letters.insert(ch);
        }
    }
    return letters.size() == 26; //26 letters in the Latin alphabet
}

int main() {
    string sentence;
    cout << "Enter a string of lowercase Latin letters: ";
    getline(cin, sentence);
    cout << boolalpha << Pangram << endl;
    return 0;
}

File: 2_4/2task.go
package main
import (
	"fmt"
)
func Pangram(sentence string) bool {
	letters := make(map[rune]bool)
	for _, ch := range sentence {
		if ch >= 'a' && ch <= 'z' {
			letters[ch] = true
		}
	}
	return len(letters) == 26
}
func main() {
	var sentence string
	fmt.Print("Enter a string of lowercase Latin letters: ")
	fmt.Scanln(&sentence)
	fmt.Println(Pangram(sentence))
}